src/Content/Embed/EmbedVideo.tsx

```diff
@@ -57,11 +57,11 @@ interface EmbedVideoProps
   extends Required<Pick<APIEmbedVideo, "width" | "height">> {
   thumbnail?: APIEmbedThumbnail["url"];
   url: APIEmbedVideo["url"] | undefined;
-  proxyUrl: APIEmbedVideo["proxy_url"] | undefined;
+  proxyUrl: APIEmbedVideo["proxy_url"] | undefined | null;
```

Comment:
```suggestion
  proxyUrl?: APIEmbedVideo["proxy_url"];
```

Reply:
This is [not equivalent](https://www.typescriptlang.org/docs/handbook/advanced-types.html#optional-parameters-and-properties).

![webstorm64_SEtUWsFiS1](https://github.com/widgetbot-io/message-renderer/assets/24935987/fac0554d-9d23-4be5-92f4-1753a7f2132c)

Reply:
@JohnyTheCarrot set it to undefined and not null?

Reply:
@Yomanz what do you mean?